build.sbt

```diff
@@ -79,7 +79,7 @@ lazy val connector = (project in file("connector"))
       "aopalliance" % "aopalliance" % "1.0" % "provided",
       "org.codehaus.jackson" % "jackson-core-asl" % "1.9.13" % "provided",
       "org.codehaus.jackson" % "jackson-mapper-asl" % "1.9.13" % "provided",
-      "org.apache.arrow" % "arrow-vector" % "0.16.0",
+      "org.apache.arrow" % "arrow-vector" % "0.16.0" exclude("org.slf4j", "slf4j-api"),
```

Comment:
Is it more appropriate to mark `slf4j` as `provided` instead? Is this included by another transitive dependency and somehow not resolved consistently by SBT, or is some external distribution already including it?

Reply:
We have it as provided, unfortunately SBT still brings it from Arrow (the resolution algorithm is unlike Maven's) . This is why we need the explicit exclude.

Reply:
Yeah, now I can see that _just_ above the fold where the diff was actually shown. :)